backend/app/models/config.py

```
@@ -0,0 +1,53 @@
+from pydantic import BaseModel, Field
+from typing import Dict, List, Optional
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)